File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.java
class Solution {
    public int canBeTypedWords(String text, String brokenLetters) {
        boolean broken[] = new boolean[26];

        for (final char bl: brokenLetters.toCharArray()) {
            broken[bl - 'a'] = true;
        }

        int result = 0;
        for (String word: text.split(" ")) {
            result++;
            for (final char l: word.toCharArray()) {
                if (broken[l - 'a']) {
                    result--;
                    break;
                }
            }
        }

        return result;
    }
}

File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.py
class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:
        broken = [False] * 26
        for bl in brokenLetters:
            broken[ord(bl) - ord('a')] = True
        
        result = 0
        for word in text.split(' '):
            result += 1
            for l in word:
                if broken[ord(l) - ord('a')]:
                    result -= 1
                    break
        
        return result